metadata.rb

```diff
@@ -26,7 +26,7 @@
 Installs and configures SpamAssassin, a mail filter software to identify spam.
 EOH
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '2.1.0' # WiP
+version '2.1.0001' # WiP
```

Comment:
Please, do not change the version.

Reply:
The reason for the changed version number is to give the local copy of the cookbook priority over the community version pending the mergen request, bit I should have excluded it for the PR.

Reply:
Done